source/documentation/admin-guide/appe-oVirt_and_SSL.html.md

```diff
@@ -92,15 +92,27 @@ The internal CA stores the internally generated key and certificate in a **P12**
         SSL_CERTIFICATE=/etc/pki/ovirt-engine/certs/apache.cer
         SSL_KEY=/etc/pki/ovirt-engine/keys/apache.key.nopass
 
-10. Restart the `ovirt-provider-ovn` service:
+10. Edit the **/etc/ovirt-imageio-proxy/ovirt-imageio-proxy.conf** 
```

Comment:
missing "file:" at the end, see section 9.

Reply:
Done